### PR TITLE
Add spatial component to local quantization

### PR DESCRIPTION
**Pull request by RunDevelopment**
I choose the default value of Spatial weight such that it doesn't affect the result with Kernel radius=1 at all. 

image

### CODE REVIEW

File: backend/src/packages/chaiNNer_standard/image_filter/quantize/quantize_local.py [hunk 1]
```diff
@@ -39,6 +47,16 @@ def quantize_image(image, palette):
             channels=[3, 4],
             image_type=navi.Image(channels_as="Input0"),
         ),
+        SliderInput("Kernel Radius", minimum=1, maximum=5, default=1),
```
Comment (joeyballentine) [inline backend/src/packages/chaiNNer_standard/image_filter/quantize/quantize_local.py:50]: I would make the default 3 personally

File: backend/src/packages/chaiNNer_standard/image_filter/quantize/quantize_local.py [hunk 2]
```diff
@@ -74,18 +94,30 @@ def quantize_local_node(
     assert i_w >= r_w, "Image width must be larger than reference image width"
     assert i_w % r_w == 0, "Image width must be a multiple of reference image width"
 
+    spatial_scale = spatial_scale / 100
+    spatial_scale = spatial_scale * spatial_scale
+    img = add_xy(img, r_w * spatial_scale)
+    reference_img = add_xy(reference_img, r_w * spatial_scale)
+    c = i_c + 2
+
+    kernel_size = 2 * kernel_radius + 1
```
Comment (joeyballentine) [inline backend/src/packages/chaiNNer_standard/image_filter/quantize/quantize_local.py:103]: Oh lol nvm

File: backend/src/packages/chaiNNer_standard/image_filter/quantize/quantize_local.py [hunk 3]
```diff
@@ -39,6 +47,16 @@ def quantize_image(image, palette):
             channels=[3, 4],
             image_type=navi.Image(channels_as="Input0"),
         ),
+        SliderInput("Kernel Radius", minimum=1, maximum=5, default=1),
+        SliderInput(
```
[the remark this answers is not in the document]
Comment (RunDevelopment) [reply backend/src/packages/chaiNNer_standard/image_filter/quantize/quantize_local.py:51]: We need docs in general. If I didn't know what this node does already, I would have no idea going by the docs alone.

[the remark this answers is not in the document]
Comment (joeyballentine) [reply backend/src/packages/chaiNNer_standard/image_filter/quantize/quantize_local.py:51]: I will improve that in my PR. I just don't think I can explain this input myself very well

**Review by joeyballentine — APPROVED** [review]
👍

**Pull request merged**